### PR TITLE
Allow Atmospheric analyzers in toolbelt

### PR DESCRIPTION
**Pull request by AutumnNova**
Allows storage of atmos analyzers in toolbelts

I added an itemtrait for the atmos analyzer to allow it to be whitelisted in toolbeltcapacity.asset, the second commit in this pr seems to have reordered the atmos analyzer prefab for whatever reason

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs [hunk 1]
```diff
@@ -34,4 +34,5 @@ public class CommonTraits : SingletonScriptableObject<CommonTraits>
 	public ItemTrait Welder;
 	public ItemTrait Shovel;
 	public ItemTrait Knife;
+	public ItemTrait AtmosAnalyser;
```
Comment (vlandemart) [inline UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs]: Should it really be a common trait? I mean, I can imagine 10 types of knives and why this is a trait but can't think of the same for atmos analyzer

Comment (AutumnNova) [reply UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs]: well how else do you want me to do it?
the whitelist system for the belt uses itemtraits

Comment (vlandemart) [reply UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs]: Maybe we should instead have "Tool" trait which will allow items to be put in toolbelts/toolboxes and possibly something else

Comment (AutumnNova) [reply UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs]: the other reason i did it this way because the health analyzer was done the same way

**Pull request merged**